src/transformers/models/clap/modeling_clap.py

```diff
@@ -1166,7 +1166,9 @@ def __init__(self, config):
         self.dropout = nn.Dropout(config.hidden_dropout_prob)
         # position_ids (1, len position emb) is contiguous in memory and exported when serialized
         self.position_embedding_type = getattr(config, "position_embedding_type", "absolute")
-        self.register_buffer("position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)))
+        self.register_buffer(
+            "position_ids", torch.arange(config.max_position_embeddings).expand((1, -1)), persistent=True
```

Comment:
Just double checking this should be `persistent=True`. Assuming yes, given other buffers but most other models seem to have it as `False`

Reply:
CLAP has the `persistent=False->persistent=True` in its copied from statement, that's why. I don't want to break it accidentally so didn't touch that statement.